src/app/static/src/app/store/root/getters.ts

```diff
@@ -1,12 +1,32 @@
 import {RootState} from "../../root";
 import {Getter, GetterTree} from "vuex";
+import {Warning} from "../../generated";
 
 interface RootGetters {
     isGuest: Getter<RootState, RootState>
+    warnings: Getter<RootState, RootState>
 }
 
 export const getters: RootGetters & GetterTree<RootState, RootState> = {
     isGuest: (state: RootState, getters: any) => {
         return state.currentUser == "guest";
+    },
+
+    warnings: (getters: any, rootGetters: any, rootState: RootState) => (stepName: string) => {
```

Comment:
This function signature doesn't match definition `Getter<S, R> = (state: S, getters: any, rootState: R, rootGetters: any) => any`. You've gotten away with it due to the way you've structured the tests but suggest the following which is both correct, and consistent with `isGuest`:
```suggestion
    warnings: (state: RootState, getters: any) => (stepName: string) => {
```